src/ol/View.js

```diff
@@ -943,7 +949,8 @@ class View extends BaseObject {
       resolution: resolution,
       rotation: rotation,
       zoom: this.getZoom()
-    };
+    });
+    return state;
```

Comment:
Not clear to me why we want to retain this state (also only reviewing on a small device).

Reply:
The idea was that when waiting for a vector tile to be loaded, the resolution of the view may still change. So when the tile has finished loading, it is more likely that the instructions will be created for the correct resolution, instead of having to recreate them for a different resolution immediately.

Reply:
Actually I just removed this commit. I do see the risk that came with it. The alternative is to build instruction groups on demand, instead of in a load listener. I've implemented that in the last commit.